avd-containers/dev/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,60 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "avd-universal",
+	"build": {
```

Comment:
build and name should be uindented from one level

Reply:
this is how VSCode idents JSONs by default. What is the reason to unindent?

Reply:
It is tabs here instead of spaces it seems.